site/docs/configurable-attributes.md

```diff
@@ -531,50 +521,81 @@ sh_binary(
 )
 ```
 
-This automatically expands the `select` to the original syntax above.
+### <a name="selects-config-setting-or-group"></a>`selects.config_setting_group`
 
-### <a name="config-setting-aliasing"></a>`config_setting` Aliasing
+The
+[config_setting_group](https://github.com/bazelbuild/bazel-skylib/blob/master/docs/selects_doc.md#selectsconfig_setting_group)
+macro in [Skylib](https://github.com/bazelbuild/bazel-skylib)'s
+[`selects`](https://github.com/bazelbuild/bazel-skylib/blob/master/docs/selects_doc.md)
+module supports `OR`ing multiple `config_setting`s:
 
-If you'd like to `OR` conditions under a proper `config_setting` that any rule
-can reference, you can use a `select`able [alias](be/general.html#alias) that
-matches any of the desired conditions:
+```python
+load("@bazel_skylib//:lib.bzl", "selects")
+```
 
 ```python
-alias(
-    name = "config1_or_2_or_3",
-    actual = select({
-        # When the build matches :config1, this alias *becomes* :config1.
-        # So it too matches by definition. The same applies for :config2
-        # and :config3.
-        ":config1": ":config1",
-        ":config2": ":config2",
-        ":config3": ":config3",
-        # The default condition represents this alias "not matching" (i.e.
-        # none of the conditions that we care about above match). In this
-        # case, bind the alias to any of those conditions. By definition
-        # it won't match.
-        "//conditions:default": ":config2", # Arbitrarily chosen from above.
-    }),
+config_setting(
+    name = "config1",
+    values = {"cpu": "arm"},
+)
+config_setting(
+    name = "config2",
+    values = {"compilation_mode": "dbg"},
+)
+selects.config_setting_group(
+    name = "config1_or_2",
+    match_any = [":cond1", ":cond2"],
 )
-
 sh_binary(
     name = "my_target",
     srcs = ["always_include.sh"],
-    deps = select({
-        ":config1_or_2_or_3": [":standard_lib"],
-        ":config4": [":special_lib"],
-    }),
+    deps = select({":config1_or_2": [":standard_lib"]}),
```

Comment:
Why a select with no default?

Reply:
Added.

Reply:
It does not show up. Did you push?

Reply:
Yep. It doesn't show up at https://github.com/bazelbuild/bazel/pull/8909/files?